### ISSUE TITLE
Support for obsidian

### ISSUE THREAD

**Issue by markfaine** [retypeapp/retype#431]
Do you have support for [Obsidian](https://help.obsidian.md/Obsidian/Index)?  If not, is this planned?

**Comment by geoffreymcgill** [comment]
Which particular feature(s) of Obsidian would you like to be supported within Retype?

I was able to open the retype.com project in Obsidian and was able to edit the pages. If you have `retype watch` running, the generated website is updated immediately in the browser. 

Obsidian does not render the Retype specific components, such as Panel and Tab, but you can still create and modify the Markdown of those components directly within Obsidian.

**Comment by markfaine** [comment]
> Obsidian does not render the Retype specific components, such as Panel and Tab, but you can still create and modify the Markdown of those components directly within Obsidian.

Yes, this is it.  I would like for it to be rendered by Obsidian as it does for any other markdown.  I understand that this request is likely more suited to a feature request on Obsidian and I have submitted one, but they do have community plugins so I'm covering all my bases.   They can support it natively or you *could* support it with a community plugin.

**Comment by geoffreymcgill** [comment]
We have not yet started on an Obsidian plugin for rendering Retype components within Obsidian, but work to improve building Markdown (.md) files created by Obsidian is underway.

The first big task was supporting `[[WikiLink]]` style of page linking, which is now complete and will be included with the upcoming Retype v3.8.0 release. Retype can now build the entire [Obsidian Help](https://github.com/obsidianmd/obsidian-help/tree/master) project.

We still have a few minor Obsidian unique features and qwerks to refine, which will not be fully available until the Retype v3.9.0 release. Once we get through v3.9.0, work will begin to create a plugin for support of Retype components within Obsidian.

Hope this helps.

**Comment by geoffreymcgill** [comment]
The `v3.8.0-preview` release is now available and the final `v3.8.0` release should be published within 1-2 days.
